services/static-webserver/client/source/class/osparc/product/tours/Tours.js

```diff
@@ -0,0 +1,63 @@
+/* ************************************************************************
+
+   osparc - the simcore frontend
+
+   https://osparc.io
+
+   Copyright:
+     2022 IT'IS Foundation, https://itis.swiss
+
+   License:
+     MIT: https://opensource.org/licenses/MIT
+
+   Authors:
+     * Odei Maiz (odeimaiz)
+
+************************************************************************ */
+
+/**
+ * @asset(osparc/tours/s4llite_tours.json)
+ * @asset(osparc/tours/s4l_tours.json)
+ * @asset(osparc/tours/tis_tours.json)
+ */
+
+qx.Class.define("osparc.product.tours.Tours", {
+  type: "static",
+
+  statics: {
+    TOURS: {
+      "s4llite": {
+        fetchTours: () => osparc.product.tours.Tours.fetchTours("/resource/osparc/tours/s4llite_tours.json")
+      },
+      "s4l": {
+        fetchTours: () => osparc.product.tours.Tours.fetchTours("/resource/osparc/tours/s4l_tours.json")
+      },
+      "tis": {
+        fetchTours: () => osparc.product.tours.Tours.fetchTours("/resource/osparc/tours/tis_tours.json")
+      }
+    },
+
+    fetchTours: function(link) {
+      return new Promise((resolve, reject) => {
+        osparc.utils.Utils.fetchJSON(link)
+          .then(toursObj => {
+            const tours = Object.values(toursObj);
+            resolve(tours);
+          })
+          .catch(err => {
+            console.error(err);
+            reject();
+          });
+      });
+    },
```

Comment:
Suggestion for a more condensed syntax: (`.then(Object.values)` is equivalent to `.then(obj => Object.values(obj))`) which is equivalent to `.then(obj => { return Object.values(obj) })`)
Also removing the catch would let you treat each error separately when you call `fetchTours` (but might not be what you want).
```suggestion
    fetchTours: function(link) {
      return osparc.utils.Utils.fetchJSON(link)
        .then(Object.values)
        .catch(console.error);
    },
```